spec/views/static_pages/collect.html.erb_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe "static_pages/collect.html.erb", type: :view do
+  pending "add some examples to (or delete) #{__FILE__}"
+end
```

Comment:
Since we're not using this test, it'd be best to delete this file (and all of the other sepc/views/static_pages files that were added). That way we just keep things clean and don't have unneeded files lying around